backend/src/remoteitInstaller.ts

```diff
@@ -1,9 +1,8 @@
 import Installer from './Installer'
-import { CLI_VERSION } from './constants'
+import { RESOURCES } from './constants'
 
 export default new Installer({
```

Comment:
@mauricio0000 This is the place where you should create a `new Installer()` for each resource.
Loop over the `RESOURCES` array and create an installer for each. You might just move the whole RESOURCES constant to this file. 

Then rename this file `installers.ts` and export the array of installer instances for the resources.

Please let me know if you have any questions I'm happy to pair with you on this. Thanks!